src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-space;
+  width: $cal-width;
+  padding: 20px;
+  margin: 0 auto;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
```

Comment:
it's the general rule, that will be used for all elements inside the project, so will be better to set it using `*` selector

Reply:
done